ui/app/components/app/menu-bar/menu-bar.component.js

```diff
@@ -10,39 +10,14 @@ export default class MenuBar extends PureComponent {
     metricsEvent: PropTypes.func,
   }
 
-  static propTypes = {
-    hideSidebar: PropTypes.func,
-    sidebarOpen: PropTypes.bool,
-    showSidebar: PropTypes.func,
-  }
-
   state = { accountDetailsMenuOpen: false }
 
   render () {
     const { t } = this.context
-    const { sidebarOpen, hideSidebar, showSidebar } = this.props
     const { accountDetailsMenuOpen } = this.state
 
     return (
       <div className="menu-bar">
-        <Tooltip
-          title={t('menu')}
-          position="bottom"
-        >
-          <div
-            className="menu-bar__sidebar-button"
-            onClick={() => {
-              this.context.metricsEvent({
-                eventOpts: {
-                  category: 'Navigation',
-                  action: 'Home',
-                  name: 'Opened Hamburger',
-                },
-              })
-              sidebarOpen ? hideSidebar() : showSidebar()
-            }}
-          />
-        </Tooltip>
         <SelectedAccount />
 
         <Tooltip
```

Comment:
removes the button for opening the sidebar in popup view. looks good, makes sense